.github/workflows/volta70.yml

```diff
@@ -0,0 +1,161 @@
+            -DKokkos_DIR=$PWD/../../kokkos/install/lib64/cmake/Kokkos \
+            -DCMAKE_CXX_FLAGS="" \
+            -DCMAKE_INSTALL_PREFIX= \
+            -DKokkosKernels_ENABLE_TESTS_AND_PERFSUITE=OFF \
+            -DKokkosKernels_ENABLE_TESTS=ON \
+            -DKokkosKernels_ENABLE_PERFTESTS=ON \
+            -DKokkosKernels_ENABLE_EXAMPLES:BOOL=ON \
+            -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF \
+            -DKokkosKernels_ENABLE_TPL_ROCSPARSE=OFF \
+            -DKokkosKernels_ENABLE_TPL_ROCBLAS=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUSOLVER=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUSPARSE=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUBLAS=OFF \
+            -DKokkosKernels_INST_LAYOUTRIGHT=ON \
+            -DCMAKE_EXE_LINKER_FLAGS="" \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkosKernels_ENABLE_DOCS=OFF \
+            ..
+
+      - name: build_kokkos_kernels
+        working-directory: kokkos-kernels/build
+        run: make -j12 all
+
+      - name: test
+        working-directory: kokkos-kernels/build
+        run: ctest --output-on-failure -V --timeout 3600
+
+  PR_VOLTA70_GCC930_CLANG13_CUDA10_CUDA_LEFT_OPENBLAS_REL:
+    name: PR_VOLTA70_GCC930_CLANG13_CUDA10_CUDA_LEFT_OPENBLAS_REL
+    runs-on: [kk-env-cuda-10.1.243-openblas-0.3.20-llvm-13.0.0-latest]
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@v3
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@v3
+        with:
+          repository: kokkos/kokkos
+          ref: 4.4.00
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build,install}
+          cd kokkos/build
+          NVCC=$(which clang++)
+          cmake -DCMAKE_CXX_COMPILER=$NVCC \
+            -DCMAKE_CXX_FLAGS="-O3 -Wall -Wunused-parameter -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wuninitialized" \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=$PWD/../install \
+            -DKokkos_ENABLE_CUDA=ON \
+            -DKokkos_ARCH_VOLTA70=ON \
+            -DKokkos_ENABLE_TESTS=OFF \
+            -DKokkos_ENABLE_EXAMPLES=OFF \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DCMAKE_CXX_STANDARD=17 \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
+            ..
+
+      - name: build_and_install_kokkos
+        working-directory: kokkos/build
+        run: make -j12 install
+
+      - name: configure_kokkos_kernels
+        run: |
+          mkdir -p kokkos-kernels/{build,install}
+          cd kokkos-kernels/build
+          NVCC=$(which clang++)
+          cmake \
+            -DCMAKE_CXX_COMPILER=$NVCC \
+            -DKokkos_DIR=$PWD/../../kokkos/install/lib64/cmake/Kokkos \
```

Comment:
We should shift to using `Kokkos_ROOT` , part of a follow up since this is already tested